test/puppetlabs/puppetdb/http/facts_test.clj

```diff
@@ -590,8 +591,8 @@
 (deftestseq ^{:postgres false} two-database-fact-query-config
   [[version endpoint] facts-endpoints]
 
-  (let [read-db-config (create-db-map)
-        write-db-config (create-db-map)
+  (let [read-db-config (create-hsqldb-map)
+        write-db-config (create-hsqldb-map)
```

Comment:
this was hsqldb only before because our testing didn't support using multiple pg databases. If that's supported now, we should flip it because we actually don't allow separate read/write dbs on hsql.